### PR TITLE
run credscan and Polichek  as its own pipeline

### PR DESCRIPTION
**Pull request by p3dr0rv**
The CredScan task is a repo wide assessment, it doesn't need to run as a task under each pipeline within a repo it can run once as its own pipeline...once per each repo.

Update credscan, policheck and publish results to v3

Related PR:
AzureAD/microsoft-authentication-library-common-for-android/pull/1437
AzureAD/ad-accounts-for-android/pull/1609
AzureAD/azure-activedirectory-library-for-android/pull/1617
Pipeline
https://identitydivision.visualstudio.com/Engineering/_build?definitionScope=%5CAndroid%5CPull%20Request%20Validations%5CCredScan

### CODE REVIEW

**Comment by p3dr0rv** [comment]
Note: CredScan pipeline will fail, it is pointing to an old template once that  common is merged this will succeed

File: azure-pipelines/vsts-releases/msal-vsts-release.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # File: azure-pipelines\vsts-releases\msal-vsts-release.yml
-# Description: Publish msal to maven central (sonatype)
+# Description: Publish msal to local feed
```
Comment (shahzaibj) [inline azure-pipelines/vsts-releases/msal-vsts-release.yml:2]:
```suggestion
# Description: Publish msal to internal feed
```

File: azure-pipelines/vsts-releases/msal-vsts-release.yml [hunk 2]
```diff
@@ -9,20 +9,9 @@ name: $(date:yyyyMMdd)$(rev:.r)
...
-    type: github
-    name: AzureAD/microsoft-authentication-library-common-for-android
-    ref: dev
-    endpoint: ANDROID_GITHUB
-
 jobs:
-- job: Phase_1
-  displayName: Phase 1
+- job: vsts_release
+  displayName: Publish msal to local feed
```
Comment (shahzaibj) [inline azure-pipelines/vsts-releases/msal-vsts-release.yml:14]:
```suggestion
  displayName: Publish msal to internal feed
```

**Review by shahzaibj — APPROVED** [review]

File: azure-pipelines/pull-request-validation/credscan.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# File: azure-pipelines\pull-request-validation\credscan.yml
```
Comment (iamgusain) [inline azure-pipelines/pull-request-validation/credscan.yml:1]: >credscan [](http://example.com/codeflow?start=48&length=8)

As this does more than credscan, maybe we can rename it to "compliance-checks.yml" or "security-checks.yml"

Comment (p3dr0rv) [reply azure-pipelines/pull-request-validation/credscan.yml]: c5a7ded616ea40b9e9b671114777f99b2ad5a175

**Pull request merged**